test/langtools/tools/javac/processing/model/util/elements/TestIsAutomaticMod.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2006, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
2006 can be removed

Reply:
I copied a file from 2006 and then edited the result, hence I started with the 2006 year.